src/components/FileImporter.js

```diff
@@ -90,6 +91,23 @@ class FileImporter extends Nimiq.Observable {
             return;
         }
 
+        // Check that the found QR code encodes a Nimiq secret
+        try {
+            // Make sure it is base64
+            const bytes = Nimiq.BufferUtils.fromBase64(decoded.substr(0, 2) === '#2' ? decoded.substr(2) : decoded);
+
+            // Make sure the data size is correct
+            if (bytes.byteLength !== KeyStore.ENCRYPTED_SECRET_SIZE
+                && bytes.byteLength !== KeyStore.ENCRYPTED_SECRET_SIZE_V2) {
+                throw new Error('Invalid encrypted secret size');
+            }
+        } catch (error) {
```

Comment:
Why via try ... catch?
What other exception are you expecting other than the one you're throwing yourself?

Reply:
The one from the base64 decoder.

Reply:
Could have amended that comment to the previous commit, but ok

Reply:
I'm squashing it on merge, ok? ;)